conjure-java-core/src/integrationInput/java/com/palantir/product/AsyncRequestProcessingTestService.java

```diff
@@ -18,41 +19,50 @@
 public interface AsyncRequestProcessingTestService {
     @GET
     @Path("async/delay")
+    @ConjureClientEndpoint(path = "/async/delay", method = "GET")
```

Comment:
While we're here, any chance we can flip the order to more closely match a request line e.g. `GET /async/delay`

Reply:
done